src/ir/enum_ty.rs

```diff
@@ -71,16 +71,20 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
I like to avoid unwrapping when it's easy to do, because it makes the code here less intertwined with code elsewhere; if enum_val_signed changes to return None for some other reason, that's a future bug avoided.  I also think once #128 lands, it'll look nice and clean.  

I'm pretty new at rust though, and brand new to rust-bindgen, so if I'm missing some bigger picture reason why it's worth keeping the duplicate check, I'm happy to change it.